api-extractor.json

```diff
@@ -27,6 +27,9 @@
     },
 
     "extractorMessageReporting": {
+      "ae-internal-missing-underscore": {
+        "logLevel": "none"
```

Comment:
this has been disabled because the docs tool wants anything marked as `@internal` to be prefixed with an underscore. We don't follow this convention (nor do I really want to) so I've disabled this.